packages/trader/build/config.js

```diff
@@ -20,6 +20,7 @@ const copyConfig = base => [
     // { from: path.resolve(__dirname, '../src/root_files/sitemap.xml'), to: 'sitemap.xml', toType: 'file' },
     { from: path.resolve(__dirname, '../src/public/images/favicons/favicon.ico'), to: 'favicon.ico', toType: 'file' },
     { from: path.resolve(__dirname, '../src/public/images/favicons/**') },
+    { from: path.resolve(__dirname, '../src/public/images/common/404.png'), to: 'public/images/common' },
```

Comment:
Why do we have to copy the individual file?

Reply:
because it doesn't fit to put it as part of the other favicon folder and we don't need any other images in common